packages/expo-localization/src/Localization.types.ts

```diff
@@ -79,6 +79,12 @@ export type Locale = {
    * `'en'`, `'es'`, `'pl'`.
    */
   languageCode: string | null;
+  /**
+   * An [ISO 15924](https://en.wikipedia.org/wiki/ISO_15924) 4-letter script code. On Android, it may be an empty string if none is defined on Web can be `null`.
```

Comment:
Can we return `null` instead of an empty string on Android? It would make the API much easier to use.

Reply:
Updated in [06f047e](https://github.com/expo/expo/pull/34984/commits/06f047e8537f2d53af5df539d46405ec19765745)
<img width="209" alt="image" src="https://github.com/user-attachments/assets/7b400e13-f7d2-46ee-af4a-9c3f0cf21b34" />